Add User Account dropin docs (#104)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/user-account/initialization.mdx

---

> title: User Account initialization

Reviewer: ```suggestion
title: User account initialization
```

> description: Learn how to configure initializer for the User Account drop-in.

Reviewer: ```suggestion
description: Learn how to configure initializer for the user account drop-in component.
```

---

import OptionsTable from '@components/OptionsTable.astro';

> The User Account drop-in does not require any special data or specific parameters to be initialized. To initialize the drop-in, you must import [`/scripts/initializers/account.js`](https://github.com/hlxsites/aem-boilerplate-commerce/blob/develop/scripts/initializers/account.js) into the appropriate file.

Reviewer: ```suggestion
The user account drop-in component does not require any special data or specific parameters to be initialized. To initialize the drop-in, you must import [`/scripts/initializers/account.js`](https://github.com/hlxsites/aem-boilerplate-commerce/blob/develop/scripts/initializers/account.js) into the appropriate file.
```

```js
import { initializers } from '@dropins/tools/initializer.js';
import { initialize } from '@dropins/storefront-account/api.js';
import { initializeDropin } from './index.js';
import { fetchPlaceholders } from '../aem.js';

await initializeDropin(async () => {
  const labels = await fetchPlaceholders();

  const langDefinitions = {
    default: {
      ...labels,
    },
  };

  return initializers.mountImmediately(initialize, { langDefinitions });
})();
```

> The `langDefinitions` property defines and registers language packs for the drop-in. This allows you to provide localized text for different languages in your application.

Reviewer: ```suggestion
The `langDefinitions` property defines and registers language packs for the user account component. This allows you to provide localized text for different languages in your application.
```

## Configuration options

> The User Account drop-in initializer accepts the following property:

Reviewer: ```suggestion
The user account component initializer accepts the following property:
```
